### ISSUE TITLE
Snackbar text colour doesn't match accessibility criteria

### ISSUE THREAD

**Issue by davwheat** [iamhosseindhv/notistack#195]
## Expected Behavior

The text should contrast the snackbar background to match [accessibility criteria](https://webaim.org/resources/contrastchecker/).

## Current Behavior

The contrast ratio between the snackbar text and background doesn't meet accessibility requirements.

| Snackbar | Issues                                 | Link                                                                      |
|----------|----------------------------------------|---------------------------------------------------------------------------|
| Warning  | Fails *ALL* accessibility requirements | https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FFA000 |
| Success  | Fails 3/5 accessibility requirements   | https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=43A047 |
| Info     | Fails 3/5 accessibility requirements   | https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=2979FF |
| Error    | Fails 1/5 accessibility requirements   | https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=D32F2F |

## Solutions

Error's background should become `#AB2323`
Success's background should become `#3CAA41`
Info's background should become `#4F95FF`

All snackbar text (except error) should become `#000000`

These changes stay as close to the original colours as possible while meeting accessibility requirements for the web.

**Comment by MareoRaft** [comment]
Are you asking the author to change the default colors, or are you asking how you can [customize the colors yourself](https://iamhosseindhv.com/notistack/demos#custom-variants)?

**Comment by davwheat** [comment]
Default colours.

The web should be accessible by default, which this is not.

**Comment by iamhosseindhv** [comment]
I understand that snackbars should be accessible natively, but suggestions made cause snackbars to be visually displeasing. I'll leave this up to individual developers until we find combinations of `backgroundColor` and text `color` that are accessible and visually appealing.

<img width="508" alt="Screenshot 2019-12-01 at 18 17 58" src="https://user-images.githubusercontent.com/26996372/69918232-9871c000-1467-11ea-96e6-fb4b1282ef30.png">
<img width="494" alt="Screenshot 2019-12-01 at 18 18 23" src="https://user-images.githubusercontent.com/26996372/69918233-9871c000-1467-11ea-8158-eb69dfa02e65.png">
